### PR TITLE
Make the title and logo templated

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk puiterwijk@redhat.com

### CODE REVIEW

File: bin/yaml2data.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+""" Convert our apps.yaml file to JSON, to stdout. """
```
Comment (ralphbean) [inline bin/yaml2data.py]: docstring is wrong.  copied.

what is this file for?

File: bin/setup.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+""" Create files from templates. """
+
+from yaml2data import get_data
+
+
+if __name__ == '__main__':
+    d = get_data()
+    with open('index.html.in', 'r') as f:
```
Comment (ralphbean) [inline bin/setup.py]: This requires ``index.html.in`` to exist in cwd?  How about taking it as a command line argument?  Or reading its contents from stdin?  Either would be fine.

File: bin/setup.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
```
Comment (ralphbean) [inline bin/setup.py]: Rename this script to something more description, like ``compile-template.py``

File: bin/setup.py (hunk 2 above)
Comment (puiterwijk) [inline bin/setup.py]: yaml2json also assumes cwd

**Comment by ralphbean** [comment]
OK.  I understand now.

So, a couple more things.  It doesn't make sense for ``yaml2data.py`` to be a script anymore.  It should have its shebang removed since running it directly doesn't do anything anymore.

Furthermore, it doesn't make sense for it to be in a ``bin/`` directory.  It should probably go in a ``lib/`` directory and:

- possibly be installed into site-packages with setuptools.. and with a better name than yaml2data.  Maybe ``apps_fp_o`` instead?
- Or.. the scripts still in ``bin/`` could use the ``site`` module to add ``../lib/`` to their pythonpath before trying to import it.  Will this all work when they're installed via rpm?

**Comment by pypingou** [comment]
@puiterwijk ping?

**Comment by ralphbean** [comment]
@puiterwijk ping?

**Comment by ralphbean** [comment]
Closing this since there hasn't been any action on it in a while.

The branch will still be there if we want to re-open it at any point.

**Pull request closed without merging**